applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.h

```diff
@@ -34,6 +36,10 @@ struct ElementalData{
     array_1d<double, TNumNodes> N;
 };
 
+typedef Node < 3 > NodeType;
```

Comment:
I think you will need this after the previous changes:
```suggestion
typedef Node < 3, Dof<double> > NodeType;
```